tests/test_login_user_correct_credentials.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+from src.pages.home_page import Home
+import time
+
+
+@pytest.mark.usefixtures("web_browser")
+class TestLoginUserWithCorrectCreds:
+    def test_home_page_visible(self, web_browser, browser):
```

Comment:
```python
    def test_home_page_visible(self, web_browser, browser):
        home_page = Home(web_browser)
        assert "Automation" in home_page.home_title_text(), "Home page is not displayed!"
``` 
Optional: the test repeated multiple time in different files - you can separate it 1 time and include this check in other test scenarios

Reply:
ok, it is done